llm/data_gemma/baseline.py

```diff
@@ -0,0 +1,45 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Basic Flow."""
+
+from data_gemma import base
+
+
+class BaselineFlow(base.Flow):
```

Comment:
Why not putting all the Flow class in flow.py?

Reply:
Because RIG / RAG flow classes are somewhat long.  So thought different file is more manageable.  Anyway end-users will access all these flows as data_gemma.BaselineFlow, etc.